Automattic/wp-calypso
# wpcalypso environment: Turn on themes/plugin-bundling and signup/seller-upgrade-modal flags

#### Proposed Changes

* wpcalypso environment: Turn on themes/plugin-bundling and signup/seller-upgrade-modal flags

#### Testing Instructions

* Wait until we have thriving-artist softlaunched https://github.com/Automattic/wp-calypso/issues/67680
* Turn on the seller-upgrade-modal flag in the local development environment
```diff
diff --git a/config/development.json b/config/development.json
index 85cc5cbab1..0b3d09bb06 100644
--- a/config/development.json
+++ b/config/development.json
@@ -161,7 +161,7 @@
                "signup/inline-help": false,
                "signup/professional-email-step": false,
                "signup/reskin": true,
-               "signup/seller-upgrade-modal": false,
+               "signup/seller-upgrade-modal": true,
                "signup/site-vertical-step": true,
                "signup/social": true,
                "signup/standard-theme-v13n": true,
```
* The `themes/plugin-bundling` flag is already on in local dev
* Visit http://calypso.localhost:3000/
* Ensure the theme showcase and the design picker are in a usable, testable state
* Choosing thriving artist should send you through the new woocommerce install flow
